common/SmokeTests/smoke-tests.yml

```diff
@@ -2,7 +2,7 @@ variables:
   - template: /eng/pipelines/templates/variables/globals.yml
 
 jobs:
-  - template: /eng/pipelines/templates/jobs/smoke.tests.yml
+  - template: /eng/pipelines/templates/jobs/new.smoke.tests.yml
```

Comment:
Did you want to fix this to just be smoke.tests.yml before this merge or did you plan to clean that up afterwards?

Reply:
This will be changed in follow-up PR